core/src/main/java/io/th0rgal/oraxen/api/events/custom_block/stringblock/OraxenStringBlockDropLootEvent.java

```diff
@@ -0,0 +1,27 @@
+package io.th0rgal.oraxen.api.events.custom_block.stringblock;
+
+import io.th0rgal.oraxen.api.events.custom_block.OraxenCustomBlockDropLootEvent;
+import io.th0rgal.oraxen.mechanics.provided.gameplay.custom_block.CustomBlockMechanic;
+import io.th0rgal.oraxen.mechanics.provided.gameplay.custom_block.stringblock.StringBlockMechanic;
+import io.th0rgal.oraxen.utils.drops.DroppedLoot;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+public class OraxenStringBlockDropLootEvent extends OraxenCustomBlockDropLootEvent {
+    public OraxenStringBlockDropLootEvent(@NotNull CustomBlockMechanic mechanic, @NotNull Block block, Player player, @NotNull List<DroppedLoot> loots) {
```

Comment:
should take a stringblock mechanic directly

Reply:
Changed. On that note, OraxenStringBlockBreakEvent also takes in CustomBlockMechanic.

Reply:
Feel free to patch that aswell, probably an oversight from copy/pasting the events originally